doc/source/getting_started/launcher.rst

```diff
@@ -0,0 +1,228 @@
+
+.. code:: console
+
+    /usr/ansys_inc/v241/ansys/bin/ansys211 -grpc
+
+This starts up MAPDL in gRPC mode, and MAPDL should output:
+
+.. code:: output
+
+     Start GRPC Server
+
+     ##############################
+     ### START GRPC SERVER      ###
+     ##############################
+
+     Server Executable   : MapdlGrpc Server
+     Server listening on : 0.0.0.0:50052
+
+You can configure the port that MAPDL starts on with the ``-port`` argument.
+For example, you can start the server to listen for connections at 
+port 50005 with:
+
+.. code:: console
+
+    /usr/ansys_inc/v241/ansys/bin/ansys211 -port 50005 -grpc
+
+
+.. _connect_grpc_madpl_session:
+
+Connect to the local MAPDL instance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An MAPDL gRPC server can be connected to from the same host by using:
+
+.. code:: pycon
+
+    >>> from ansys.mapdl.core import Mapdl
+    >>> mapdl = Mapdl()
+
+This assumes that your MAPDL service is running locally on the default IP address 
+(``127.0.0.1``) and on the default port (``50052``).
+
+You can also use the :func:`launch_mapdl() <ansys.mapdl.core.launcher.launch_mapdl>` to connect to an already launched MAPDL instance by setting the ``start_instance`` argument to ``False``.
+
+.. code:: pycon
+
+    >>> from ansys.mapdl.core import launch_mapdl
+    >>> mapdl = launch_mapdl(start_instance=False)
+
+If you are connecting to an MAPDL Docker image, the procedure is the same.
+Just make sure that you specify the mapped port instead of the internal MAPDL port.
+For more information, see :ref:`pymapdl_docker`.
+
+
+.. _connect_grpc_remote_madpl_session:
+
+Connect PyMAPDL to a remote MAPDL instance
+------------------------------------------
+
+If you want to connect to a **remote** MAPDL instance, you must know the IP 
+address of that instance, so you can connect to it.
+For example, if on your local network at IP address ``192.168.0.1`` there is a
+computer running MAPDL on the port 50052, you can connect to it with:
+
+.. code:: pycon
+
+    >>> mapdl = Mapdl("192.168.0.1", port=50052)
+
+Alternatively you can use a hostname:
+
+.. code:: pycon
+
+    >>> mapdl = Mapdl("myremotemachine", port=50052)
+
+Note that you must have started an MAPDL instance in gRPC mode on the computer with
+the mentioned IP address/hostname for this to work, since PyMAPDL cannot launch remote instances.
+
+
+Setting MAPDL location in PyMAPDL
+---------------------------------
+
+To run PyMAPDL must know the location of the MAPDL binary. 
+Most of the time this can be automatically determined, but
+the location of MAPDL must be provided for non-standard installations.
+When running for the first time, PyMAPDL requests the
+location of the MAPDL executable if it cannot automatically find it.
+
+You can test your installation of PyMAPDL and set it up by running
+the :func:`launch_mapdl() <ansys.mapdl.core.launcher.launch_mapdl>` function:
```

Comment:
Is this a function or method? Dominick taught me that functions aren't part of class but methods are. I used method earlier--so it may be wrong or this may be wrong?!

Reply:
ahahha... it is a function indeed. :)

Dominik told you correctly :)